mmv1/templates/terraform/examples/cloud_run_service_configuration.tf.erb

```diff
@@ -0,0 +1,156 @@
+# Example configuration of a Cloud Run service
+
+# Min instances
+# https://cloud.google.com/run/docs/configuring/min-instances
+variable "min_instances" {
```

Comment:
Would you mind hardcoding these variables in the example? I think that will make the documentation a little bit more uniform with our other examples and make the example a bit more concise. Thanks!